tests/lean/floatAbs.lean

```diff
@@ -0,0 +1,7 @@
+def main : IO Unit := do
+IO.println ((3.14159 : Float).abs);
+IO.println ((-3.14159 : Float).abs);
+IO.println ((-2 : Float).abs);
+IO.println ((0 : Float).abs);
```

Comment:
This should probably go in the compiler tests, so it is tested as both compiled and interpreted. I recommend adding it to `float.lean` next to the other float ops. Also, float ops should always be tested on infinities, NaN, and signed zero.

Reply:
I moved the tests as requested. 

I wasn't sure if there is a way to explicitly input -inf or NaN as a float, so I used 1 / 0, and 0 / 0 respectively.